ext/mysql2/extconf.rb

```diff
@@ -73,14 +74,8 @@ def asplode lib
   rpath_dir = libs
 else
   inc, lib = dir_config('mysql', '/usr/local')
-  libs = ['m', 'z', 'socket', 'nsl', 'mygcc']
```

Comment:
I've preserved this in my refactored extconf -- this only applies to Unix-like platforms (with good reason)